packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -84,6 +106,35 @@ export default class TSGenerator extends CodeGenerator {
     } as JSONSchemaToTypescriptOptions);
   }
 
+  async installer(storage: Storage, opts: InstallerOptions = {}): Promise<void> {
+    const installDir = storage.getIdentifierStorageDir();
+
+    const pkg = {
+      name: `@api/${storage.identifier}`,
+      main: './index.ts',
+    };
+
+    fs.writeFileSync(path.join(installDir, 'package.json'), JSON.stringify(pkg, null, 2));
+
+    await execa(
+      'npm',
+      ['install', ...Object.keys(this.requiredPackages), '--save', opts.dryRun ? '--dry-run' : ''].filter(Boolean),
+      { cwd: installDir }
+    ).then(res => {
+      if (opts.dryRun) {
+        (opts.logger ? opts.logger : logger)(res.command);
+        (opts.logger ? opts.logger : logger)(res.stdout);
+      }
+    });
+
+    return execa('npm', ['install', storage.getIdentifierStorageDir(), '--dry-run']).then(res => {
```

Comment:
It's supposed to but I'm just now realizing that I left in the `--dry-run` here. No wonder `@api/petstore` hasn't been getting installed. I'll fix this and then add a bunch of comments in this function to clarify what's happening because it definitely isn't clear at the moment.